test/Microsoft.AspNetCore.Authentication.Test/JwtBearerTests.cs

```diff
@@ -396,7 +396,7 @@ public class JwtBearerTests
                 {
                     OnMessageReceived = context =>
                     {
-                        context.Skip();
+                        context.None();
```

Comment:
None isn't a great name, probably need to pick something else, or add sugar for it as appropriate